Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -53,9 +53,9 @@ final class BTAnalyticsService: AnalyticsSendable {
     
     /// Sends analytics event to https://api.paypal.com/v1/tracking/batch/events/ via a background task.
     /// - Parameter event: A single `FPTIBatchData.Event`
-    func sendAnalyticsEvent(_ event: FPTIBatchData.Event) {
+    func sendAnalyticsEvent(_ event: FPTIBatchData.Event, sendImmediately: Bool) {
         Task(priority: .background) {
-            await performEventRequest(with: event)
+            sendImmediately ? await sendAnalyticsEvents(with: event) : await performEventRequest(with: event)
```

Comment:
Is there any way to combine the two methods? If not can we add some docstrings to indicate what the differences are to each method?